test/xdr/operations/operations_test.exs

```diff
@@ -0,0 +1,95 @@
+defmodule Stellar.XDR.OperationsTest do
+  use ExUnit.Case
+
+  alias Stellar.XDR.{
+    AccountID,
+    CryptoKeyType,
+    Int64,
+    MuxedAccount,
+    Operation,
+    Operations,
+    OperationBody,
+    OperationType,
+    OptionalMuxedAccount,
+    PublicKey,
+    PublicKeyType,
+    UInt256
+  }
+
+  alias Stellar.XDR.Operations.CreateAccount
+
+  describe "Operations" do
+    setup do
+      pk_key =
+        UInt256.new(
+          <<18, 27, 249, 51, 160, 215, 152, 50, 153, 222, 53, 177, 115, 224, 92, 243, 51, 242,
+            249, 40, 118, 78, 128, 109, 86, 239, 171, 232, 42, 171, 210, 35>>
+        )
+
+      source_account =
+        CryptoKeyType.new(:KEY_TYPE_ED25519)
+        |> MuxedAccount.new(pk_key)
+        |> OptionalMuxedAccount.new()
+
+      destination =
+        PublicKeyType.new(:PUBLIC_KEY_TYPE_ED25519)
+        |> (&PublicKey.new(pk_key, &1)).()
+        |> AccountID.new()
+
+      starting_balance = Int64.new(5_000_000_000)
+
+      operation_body =
+        destination
+        |> CreateAccount.new(starting_balance)
+        |> OperationBody.new(OperationType.new(:CREATE_ACCOUNT))
+
+      operation = Operation.new(source_account, operation_body)
+
+      %{
+        elements: [operation, operation],
```

Comment:
Creating an array with the same operation twice doesn't seem to be correct. A better option could be just `elements: [operation]` or `elements: [operation_1, operation_2]`

Reply:
@luishurtado definitely! changing it